suite/tests/client-interface/inline.dll.c

```diff
@@ -207,7 +207,7 @@ codegen_instrumentation_funcs(void)
         dr_log(dc, LOG_EMIT, 3, "Generated instrumentation function %s at "PFX
                ":", func_names[i], pc);
         instrlist_disassemble(dc, pc, ilists[i], dr_get_logfile(dc));
-        pc = instrlist_encode(dc, ilists[i], pc, false);
+        pc = instrlist_encode(dc, ilists[i], pc, true);
```

Comment:
Yup, that's what I did to view it, but I could not find a way to add a comment on any part of the existing file: only on the parts that were changed.  This is a not uncommon thing to want to do, and Rietveld supports it.  Maybe I missed something: did you see any way to add such a comment?